tests/test_authentication.py

```diff
@@ -140,13 +139,17 @@ def test_unknown_creds(self):
                               '^access denied: CredentialsLookupError: ')
 
     def test_alternative_credential_lookup(self):
+        # Temporarily use alternate auth
+        old_auth = self.auth
+        self.auth = AlternateAuthentication()
```

Comment:
I wasn't sure if there was any extra magic with it being attached to `self`. Will change that.